tests/Queue.test.js

```diff
@@ -0,0 +1,30 @@
+import _ from 'underscore';
+import { Queue } from '../src/classes/Queue';
+
+describe ('test the Queue class', () => {
+
+  let obj1;
+  let newQueue;
+
+  beforeEach(() => {
+
+    obj1 = {
+      x: 1,
+      y: 2,
+    };
+
+    newQueue = new Queue();
+  });
+
+  test ('should be an instance of Queue', () => {
+    expect(newQueue).toBeInstanceOf(Queue);
+  });
+
+  test ('the Queue constructor accepts the given data as argument correctly', () => {
+
+    newQueue.add(obj1);
+    let node = newQueue.pop();
+
+    expect(_.isEqual(node.data, obj1)).toBe(true);
+  });
```

Comment:
If you agree [with the comment](https://github.com/andreslopezlu/snake-utils/pull/2/files#r1699353889), we must update this test

```suggestion
  test ('the Queue constructor accepts the given data as argument correctly', () => {
    newQueue.add(obj1);
    expect(_.isEqual(newQueue.pop(), obj1)).toBe(true);
  });
```